### PR TITLE
Add `delete_timing`

### LINKED ISSUE #54: As a user, I can remove class timings of my friend's module

### PR DESCRIPTION
**Pull request by AdityaB4**
Fixes #54.

Uses the same command format as `add_timing`.

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/DeleteStudentModuleTimingCommand.java [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+        Student studentToModify = lastShownList.get(index.getZeroBased());
+
+        if (!model.doesStudentHaveModule(studentToModify, moduleCode)) {
+            throw new CommandException(MESSAGE_MODULE_NOT_FOUND);
+        }
+        // TODO: check if validation is needed
+        // if (model.doesStudentModuleTimingClash(studentToModify, moduleTiming)) {
+        //     throw new CommandException(MESSAGE_MODULE_TIMING_CLASH);
+        // }
```
Comment (blaukc) [inline src/main/java/seedu/address/logic/commands/DeleteStudentModuleTimingCommand.java]: I believe this should be a validation to check if the module timing exists in the Student

File: src/main/java/seedu/address/logic/commands/DeleteStudentModuleTimingCommand.java [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+            + PREFIX_MODULE_CODE + "MODULE CODE"
+            + PREFIX_DAY + "DAY"
+            + PREFIX_START_TIME + "START TIME"
+            + PREFIX_END_TIME + "END TIME";
+
+    public static final String MESSAGE_SUCCESS = "Deleted module timing %1$s from student: %2$s";
+    public static final String MESSAGE_MODULE_NOT_FOUND =
+            "This module does not exist in the student's contact in address book";
+    public static final String MESSAGE_MODULE_TIMING_CLASH =
+            "There is a clash in module timings";
```
Comment (blaukc) [inline src/main/java/seedu/address/logic/commands/DeleteStudentModuleTimingCommand.java]: This can be removed/changed to smth else

**Review by blaukc — APPROVED** [review]
Validation should be added for the delete. I feel like validation is a feature as it changes the behaviour. Otherwise lgtm

File: src/main/java/seedu/address/logic/commands/DeleteStudentModuleTimingCommand.java (hunk 1 above)
Comment (AdityaB4) [reply src/main/java/seedu/address/logic/commands/DeleteStudentModuleTimingCommand.java]: Added this in, thanks Brandon!

**Pull request merged**